test_scripts/Resumption/Handling_errors_from_HMI/030_resume_after_unsuccessful_attempt_unexpected_disconnect.lua

```diff
@@ -0,0 +1,68 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0190-resumption-data-error-handling.md
```

Comment:
To check req-t on number of ignition cycles to restore the resumption data

Reply:
There is a req-t:
> SDL must initiate data resumption process after the app's RegisterAppInterface_request with valid hashID in case the application has been disconnected on Ignition Off not later than 3 ignition consecutive cycles before.
>
> Information: if app didn't register during 3 ign consecutive cycles -> SDL must delete app`s related data from resumption list

In our case application is re-registers in a new ignition cycle, so resumption should take place.
And after re-registration `hashId` of the application is re-calculated, so the app need to re-register in next ignition cycle with a new value.